package.json

```diff
@@ -44,33 +45,53 @@
     "which": "^2.0.0"
   },
   "scripts": {
-    "clean": "rm -rf node_modules && rm -f package-lock.json && npm install",
-    "prepare": "gulp prepublish",
-    "test": "gulp once",
-    "watch": "gulp watch",
-    "build": "gulp transpile",
-    "e2e-test": "gulp e2e-test",
-    "mocha": "mocha",
-    "coverage": "gulp coveralls",
+    "build": "rm -rf build/lib/* && babel --out-dir=build/lib lib",
+    "build-test": "rm -rf build/test/* && babel --out-dir=build/test test --copy-files",
```

Comment:
you don't need to build the tests since mocha runs w/ @babel/register. just run mocha against test/ instead of build/test

Reply:
fixed

Reply:
`npm run build` is still happening before tests. does it need to?  I suppose if there's some sort of command-line e2e test going on

Reply:
I was just not sure if mocha babel plugin also covers that. My assumption was it only automatically compiles tests, but not src. Is that assumption right?

Reply:
for posterity

![image](https://user-images.githubusercontent.com/924465/186013304-3dd4af8a-838a-4167-a029-eb5e274ab123.png)